components/BottomMenuBar.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
+import { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+function getIconName(routeName: string) {
+  switch (routeName) {
+    case 'Schedule':
+      return 'calendar-outline';
+    case 'Recently Played':
+      return 'albums-outline';
+    case 'About':
+      return 'information-circle-outline';
+    case 'Home':
+      return 'home-outline';
+    default:
+      return 'ellipse-outline';
+  }
+}
+
+export default function BottomMenuBar({ state, navigation }: BottomTabBarProps) {
+  const activeIndex = state.index;
+
+  return (
+    <View style={styles.container}>
+      {state.routes.map((route, idx) => {
+        const focused = idx === activeIndex;
+        const iconName = getIconName(route.name);
+
+        return (
+          <TouchableOpacity
+            key={route.key}
+            style={styles.tab}
+            onPress={() => navigation.navigate(route.name)}
```

Comment:
not important now - down the road we'll want to wrap this in a usecallback to prevent excess re-renders  - it might require extracting the icons to their own component, so we can have a useCallback per component vs a dictionary/map. But it's a shallow tree so re-render isn't going to make much too much impact on its own